### DISCUSSION TITLE
github desktop

### DISCUSSION THREAD

**Discussion by Noeqt** [community/community#110721] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Why i cant download github desktop on my windows 10.It says there was an error while installing the application.
[setup..txt](https://github.com/community/community/files/14468730/setup.txt)

**Comment by AdRohal** [comment]
1- Check the Antivirus and Firewall, Sometimes, security software can interfere with the installation.
2- Check if you download the latest version
3- if you have the installer click "Run as Administrator"
4- Check if you have a free disk space 
I hope that gonna help~!!

**Comment by steveward** [comment]
@Noeqt Unfortunately I'm not seeing anything in the Squirrel log file that is showing errors related to GitHub Desktop. Do you see anything when you attempt to install the app (like a splash screen showing the installation is starting)?
